frontend/src/components/hashTag/HashTagSearchBox/HashTagSearchBox.tsx

```diff
@@ -1,69 +1,91 @@
-import { useState } from 'react';
+import React, { FormEvent, FormEventHandler, useState } from 'react';
 
+import HashTagClickSearchBox from '@/components/hashTag/HashTagClickSearchBox/HashTagClickSearchBox';
 import * as S from '@/components/hashTag/HashTagSearchBox/HashTagSearchBox.styles';
 
 export interface HashTagSearchBoxProps {
 	targets: { name: string; isChecked: boolean }[];
 	setTargets: React.Dispatch<React.SetStateAction<{ name: string; isChecked: boolean }[]>>;
 }
 
-const HashTagSearchBox = ({ targets, setTargets }: HashTagSearchBoxProps) => {
-	const [isOpen, setIsOpen] = useState(false);
+type HashTagSearchResultType = { name: string; isChecked: boolean }[];
```

Comment:
이런 경우는 뒤에 []은 빼고 사용하는 측에서 []를 붙히는것이 훨씬 좋다고 하더라고요.

```suggestion
type HashTagItem = { name: string; isChecked: boolean };
useState<HashTagItem[]>
```
이런느낌으로요! 사용하는측에서 이것이 리스트라는것을 알 수 있는것이 더 좋은방법이라고 합니다!

Reply:
기존에 prop에 선언된 타입을 재활용하는 것으로 수정하였습니다